rpcs3/stdafx.h

```diff
@@ -7,9 +7,6 @@
 #include "Utilities/File.h"
 #include "util/logs.hpp"
 #include "util/shared_ptr.hpp"
-#include "util/typeindices.hpp"
-#include "util/fixed_typemap.hpp"
```

Comment:
Only fixed_typemap.hpp uses RTTI to memoize object names. Other two headers can remain.

Reply:
Alternatively, we can remove RTTI from it, since it only to print unreadable crap in logs for debugging purposes long forgotten.

Reply:
I went with removing the name.
If you would prefer me to replace it with like the address of object (in logging for example), let me know.
Otherwise I could remove completely the 'Report' tracking ^^

Reply:
Now reporting is almost useless, but it's okay, thanks. It can be left in this state for now.

Reply:
Now reporting is almost useless, but it's okay, thanks. It can be left in this state for now.